src/components/ResourceTypeSelector/ResourceTypeSelector.js

```diff
@@ -41,21 +40,28 @@ const ResourceTypeSelector = ({
   selectResourceTypeAction,
   selectedResourceType,
 }) => (
-  <ScrollView style={styles.root} horizontal showsHorizontalScrollIndicator={false}>
-    {Object.entries(resourceTypeFilters).map(([resourceType, filterOpen]) => {
-      if (filterOpen) {
-        return (
-          <CategoryButton
-            key={resourceType}
-            resourceType={resourceType}
-            selectedResourceType={selectedResourceType}
-            selectResourceTypeAction={selectResourceTypeAction}
-          />
-        );
-      }
-      return null;
-    })}
-  </ScrollView>
+  <View>
+    <ScrollView
+      style={styles.root}
+      horizontal
+      showsHorizontalScrollIndicator={false}
+      contentContainerStyle={styles.contentContainerStyle}
+    >
+      {Object.entries(resourceTypeFilters).map(([resourceType, filterOpen]) => {
+        if (filterOpen) {
+          return (
```

Comment:
`Object.entries(...)` could be refactored to include a chained `.filter(..)`,  and this condition removed